baseapp-chats/baseapp_chats/base.py

```diff
@@ -29,6 +29,7 @@ class AbstractBaseChatRoom(TimeStampedModel, RelayModel):
     last_message_time = models.DateTimeField(null=True, blank=True, default=timezone.now)
     participants_count = models.IntegerField(default=0)
     messages_count = models.IntegerField(default=0)
+    is_group = models.BooleanField(default=False)
```

Comment:
we're talking that maybe having a `room_type` would be better here, do you still agree @nossila ?

Reply:
@anicioalexandre I'm fine with this BooleanField right now. I don't see us having other types, and its easy to change latter.